### ISSUE TITLE
ImportError: cannot import name 'InstaBot' from 'instabot.bot'

### ISSUE THREAD

**Issue by Deret007** [instabot-py/instabot.py#2393]
Check this or we will delete your issue. (fill in the checkbox with an X like so: [x])
- [x] I have searched for other issues with the same problem or similar feature requests. 

#### Select one:
- [x] Bug
- [ ] Feature Request
- [ ] Technical Help (Please don't create an issue, join the facebook group or telegram channel)
- [ ] Other

#### Environment
- [x] Are you hosting the bot on a VPS or other Cloud hosting?
- [ ] Are you running the code on your phone 
- [ ] If so, have you tried running the code on your local machine?

```
Include more details about the environment
```

#### Operating System? (include version)
- [ ] macOS
- [ ] Windows
- [ ] Rasbian Linux Raspberry PI (include flavour)
- [x] Linux (include flavour)
- [ ] Android (include terminal)

```
include the output of 'uname -a'
```

#### Python Version Requirement
- [x] I am using Python 3.6 or above

#### Exact Python Version?
Python3.8
Include here the response of 'python --version' or 'python3 --version'
```

#### Pip Version?
```
Include here the response of 'pip --version' or 'pip3 --version'
```

#### Instabot Version?
```
Include here the response of 'instabot-py --version'
```

#### How you use the instabot
- [ ] instabot-py -c my-configfile.yml (my-configfile.yml is an example you may have a different file )
- [ ] instabot-py --login LOGIN --password PASSWORD
- [ ] instabot-py 
- [ ] example.py (Deprecated, please install newer version) 


#### Description of your issue

```
Include the contents of the error.log here (if applicable). < REPLACE THIS TEXT
```

#### Config

```
Ensure that you specify the configuration file in the command line.
Ensure that the default file instabot.config.yml contains only common settings if you use multiple accounts.
Include the contents of your config here (instabot-py -c config.yml --dump) (REMOVE YOUR USERNAME AND PASSWORD!) (if applicable). < REPLACE THIS TEXT

```
After upgrading on new version I get this error

Traceback (most recent call last):
  File "/usr/local/bin/instabot-py", line 6, in <module>
    from instabot.__main__ import main
  File "/usr/local/lib/python3.8/site-packages/instabot/__init__.py", line 1, in <module>
    from .bot import InstaBot
ImportError: cannot import name 'InstaBot' from 'instabot.bot' (/usr/local/lib/python3.8/site-packages/instabot/bot/__init__.py)

**Comment by MeisterEder1987** [comment]
Have the same problem. Is there already a solution?

**Comment by yankerssnh** [comment]
Same problem :(

**Comment by TheProOfficial** [comment]
**try this:**

import instabot
bot = instabot.Bot()

**Comment by Viktor19931** [comment]
same problem ( any solutions ?

**Comment by pantocrator77** [comment]
same here!

**Comment by nickgryg** [comment]
It was fixed long time ago starting from version 0.5.12.
